resolwe/flow/models.py

```diff
@@ -361,8 +361,10 @@ class Meta(BaseModel.Meta):
     descriptor = JSONField(default={})
 
     def save(self, *args, **kwargs):
+        force_render_descriptor = kwargs.pop('force_render_descriptor', False)
+        defer_render_descriptor = kwargs.pop('defer_render_descriptor', False)
 
-        if not self.pk:  # create
+        if force_render_descriptor or (not defer_render_descriptor and not self.pk):  # create
```

Comment:
Is it on purpose to (not) set default values depending on ``*_render_descriptor`` values?